clients/src/main/resources/common/message/BeginQuorumEpochResponse.json

```diff
@@ -17,25 +17,36 @@
   "apiKey": 53,
   "type": "response",
   "name": "BeginQuorumEpochResponse",
-  "validVersions": "0",
-  "flexibleVersions": "none",
+  // Version 1 adds flexible versions and leader endpoint (KIP-853)
+  "validVersions": "0-1",
+  "flexibleVersions": "1+",
   "fields": [
     { "name": "ErrorCode", "type": "int16", "versions": "0+",
       "about": "The top level error code."},
     { "name": "Topics", "type": "[]TopicData",
       "versions": "0+", "fields": [
-      { "name": "TopicName", "type": "string", "versions": "0+", "entityType": "topicName",
-        "about": "The topic name." },
-      { "name": "Partitions", "type": "[]PartitionData",
-        "versions": "0+", "fields": [
-        { "name": "PartitionIndex", "type": "int32", "versions": "0+",
-          "about": "The partition index." },
-        { "name": "ErrorCode", "type": "int16", "versions": "0+"},
-        { "name": "LeaderId", "type": "int32", "versions": "0+", "entityType": "brokerId",
-          "about": "The ID of the current leader or -1 if the leader is unknown."},
-        { "name": "LeaderEpoch", "type": "int32", "versions": "0+",
-          "about": "The latest known leader epoch"}
-      ]}
-    ]}
+        { "name": "TopicName", "type": "string", "versions": "0+", "entityType": "topicName",
+          "about": "The topic name." },
+        { "name": "Partitions", "type": "[]PartitionData",
+          "versions": "0+", "fields": [
+            { "name": "PartitionIndex", "type": "int32", "versions": "0+",
+              "about": "The partition index." },
+            { "name": "ErrorCode", "type": "int16", "versions": "0+"},
+            { "name": "LeaderId", "type": "int32", "versions": "0+", "entityType": "brokerId",
+              "about": "The ID of the current leader or -1 if the leader is unknown."},
+            { "name": "LeaderEpoch", "type": "int32", "versions": "0+",
+              "about": "The latest known leader epoch"}
+          ]
+        }
+      ]
+    },
+    { "name": "NodeEndpoints", "type": "[]NodeEndpoint", "versions": "1+", "taggedVersions": "1+", "tag": 0,
+      "about": "Endpoints for all leaders enumerated in PartitionData", "fields": [
+        { "name": "NodeId", "type": "int32", "versions": "1+",
+          "mapKey": true, "entityType": "brokerId", "about": "The ID of the associated node" },
+        { "name": "Host", "type": "string", "versions": "1+", "about": "The node's hostname" },
+        { "name": "Port", "type": "int32", "versions": "1+", "about": "The node's port" }
```

Comment:
This again seems like it should be `int16`?

Reply:
Fixed to uint16.